examples/network-bandwidth-example.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: pingcap.com/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-bandwidth-example
+  namespace: chaos-testing
+spec:
+  action: bandwidth
+  mode: one
+  selector:
+    labelSelectors:
+      "app.kubernetes.io/component": "tikv"
+  bandwidth:
+    rate: 100kbps
+    limit: 100
```

Comment:
What will happen if we set this value to "0"?

Reply:
Good question, it will fail to apply qdisc as limit is required for tbf.